lfa_lab/core/NdRange.i

```diff
@@ -19,7 +19,7 @@
 
 // vim: set filetype=cpp:
 
-%exception PyNdRangeIterator::next {
```

Comment:
Will this change also work in Python 2 or do we need both?

Reply:
This is connected with the potential fix above.